_projects/documentation_extraction.md

```diff
@@ -16,7 +16,8 @@ mentors:
 name: "Documentation Extraction and Parsing"
 requirements:
   - "The participant should have one bugfix patch to any bear accepted."
-  - "The participant should have a basic understanding of the algorithms behind existing documentation extraction routines."
+  - "The participant should have a basic understanding of the algorithms behind existing documentation extraction
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.